assets/admin/editor-wizard/helpers.js

```diff
@@ -188,3 +189,33 @@ export const useHideEditorWizardUpsell = () => {
 		! senseiProExtension || senseiProExtension.is_activated === true
 	);
 };
+
+/**
+ * Hook to get the site slug.
+ *
+ * @return {string} Site slug.
+ */
+export const useSiteSlug = () => {
+	const { siteUrl } = useSelect( ( select ) => ( {
+		siteUrl: select( coreStore ).getSite()?.url,
+	} ) );
+
+	// Return the site slug without the protocol and trailing slash.
+	return siteUrl?.split( '/' ).pop();
+};
+
+/**
+ * Check if the current environment is WPCOM.
+ *
+ * @return {boolean} True if running on WPCOM.
+ */
+export const isWpcom = () => {
+	// Check for Jetpack connection
+	if ( window?.JetpackScriptData?.connection?.connectionStatus?.isActive ) {
+		return true;
+	}
```

Comment:
This logic will cause `isWpcom` to return true on self-hosted Jetpack sites too I think. Which will probably cause the WPCOM url to open instead of the Sensei-LMS one. That's probably not expected.

Reply:
You are right. I overlooked it...
I'll try another approach. Thanks!

Reply:
Hey @Imran92,

I have the logic and found the better field to check, which is only presented in the WPCOM env. Please re-review when you find some time.